### ISSUE TITLE
jvm crash when call .close()

### ISSUE THREAD

**Issue by peidong-hu** [vinhkhuc/JFastText#27]
Hi when my java code calls .close() before deploy the bean in spring container, I got the jvm crashed with the error as blow,
==============
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fe80add9d1c, pid=4700, tid=0x00007fe75f8ec700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_144-b01) (build 1.8.0_144-b01)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.144-b01 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x82d1c]  cfree+0x1c
#
# Core dump written. Default location: /home/peidong/Downloads/cuba/sts-bundle/sts-3.9.0.RELEASE/core or core.4700
#
# An error report file with more information is saved as:
# /home/peidong/Downloads/cuba/sts-bundle/sts-3.9.0.RELEASE/hs_err_pid4700.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

**Comment by peidong-hu** [comment]
[hs_err_pid4700.log](https://github.com/vinhkhuc/JFastText/files/1484884/hs_err_pid4700.log)

The jvm error log attached here too.

Thanks,
Peidong

**Comment by thunderstumpges** [comment]
Actually I *think* I have resolved this following the recommendation over at #29 

The master at fastText has moved way past what it was on Nov 7, and the head of master did NOT work for me, however picking up commit c5cb6b2d0e295e58cfa827e38d2e9b1e0cbe09e4 (fasttext commit on Nov7) has gotten the tests to run and `mvn package` to complete successfully!

Now if we could get a maintainer to check out issue #29 and maybe pull in the fix!

**Comment by peidong-hu** [comment]
Nope, I used a workaround to wrap it as web service and deployed with
docker as back end microservice. And then instead of close the jfastText, I
just restart docker container to avoid the jfastText's crash impacts all my
application.

Peidong
